dev/com.ibm.ws.cdi.interfaces/src/com/ibm/ws/cdi/package-info.java

```diff
@@ -12,8 +12,7 @@
  * @version 1.0
  */
 @org.osgi.annotation.versioning.Version("1.0")
-@TraceOptions(traceGroup = "JCDI")
+@TraceOptions(traceGroup = "JCDI", messageBundle = "com.ibm.ws.cdi.internal.resources.CDI")
 package com.ibm.ws.cdi;
```

Comment:
I think you need to increase the major package version as you've removed CDIContainerConfig from this package.